### PR TITLE
Show/Hide layer controls moved to title bars in step 2

### LINKED ISSUE #2051: switching layers on and off in step 2
Now we have the accordion the location of the button to switch layers on and off in step 2 is on the 'layer' tab as 'shown on map'. This makes hard work for the user to switch layers on and off. As a user I want this option to be more visible, and preferably on the title bar next to, or under, the 'X'. Also the wording 'shown on map' is not intuitive, the wording should be 'enable/disable layer'.

### PR DESCRIPTION
**Pull request by pmbohm**
fixes #2051

### CODE REVIEW

File: web-app/js/portal/details/DataCollectionDetailsPanel.js [hunk 1]
```diff
@@ -109,6 +126,17 @@ Portal.details.DataCollectionDetailsPanel = Ext.extend(Ext.Panel, {
         };
     },
 
+    _showHideLayer: function() {
+        var thisLayer= this.dataCollection.layerSelectionModel.selectedLayer;
+        console.log(this.dataCollection.layerSelectionModel);
```
Comment (jonescc) [inline web-app/js/portal/details/DataCollectionDetailsPanel.js]: do we leave these in now?

**Comment by pmbohm** [comment]
fixes #2051

**Comment by pmbohm** [comment]
@jonescc New more professional pull request up now

File: web-app/js/portal/lang/en.js [hunk 2]
```diff
@@ -228,6 +226,7 @@ OpenLayers.Lang.en = OpenLayers.Util.extend(OpenLayers.Lang.en, {
     depthLabel: 'Depth:',
     elevationLabel: 'Elevation:',
 
+    showHideTooltip: '{0} from the map',
     removeDataCollectionTooltip: 'Remove this data collection from the Subset and Download pages, the collection can be re-added from the search',
```
Comment (jonescc) [inline web-app/js/portal/lang/en.js]: Sorry should have mentioned this before - you happy with "Hide from map"  and "Show from map" as the hints?

Comment (pmbohm) [inline web-app/js/portal/lang/en.js]: Yes i'm happy because they are short and plain non techy/sciencey 
Have you got other ideas?

**Comment by jonescc** [comment]
Kind of outside the scope of the original request, but another control on the title bar makes the display more complicated - do we really need to be able to toggle layer visibility at all - they could just delete/re-add it.

File: web-app/js/portal/lang/en.js (hunk 2 above)
Comment (jonescc) [inline web-app/js/portal/lang/en.js]: "Show on map"/"Hide on map"?

**Comment by pblain** [comment]
Cramming features into the title bar sounds ugly.  Pretty sure this feature was not planned.

**Comment by pmbohm** [comment]
Here's a screen shot of the new (old) feature.
screen shot 2015-12-10 at 9 51 55 am

**Comment by bhasin85** [comment]
@pmbohm  Can we merge/close this PR?

**Pull request closed without merging**